README.md

```diff
@@ -22,6 +22,10 @@ in the `env` section of your
 to use a different major (e.g., "10" or "9.6") version. This feature
 is experimental and subject to change.
 
+## PostgreSQL fsync
+
+If you want to turn off fsync during your test, you can set `HEROKU_CI_POSTGRES_FSYNC_OFF` in your [app.json env](https://devcenter.heroku.com/articles/app-json-schema#env) section.
```

Comment:
Should this be linking to https://devcenter.heroku.com/articles/heroku-ci#environment-variables-env-key - the CI env variable docs? That's where it links to above for `POSTGRESQL_VERSION`. Seems like you if you don't put it in `environments.test.env` (ie just put it in top-level `env`) then it may get overwritten:

> When the platform applies `app.json` to an app, the values for keys defined in the applicable environment _completely replace_ the values defined in the base manifest.